packages/components/pie-tag/src/defs.ts

```diff
@@ -12,6 +12,11 @@ export interface TagProps {
      */
     isStrong?: boolean;
 
+    /**
+     * When true, sets the tag opacity to 50%
```

Comment:
same comment about the 50% reference as above

Reply:
can you please suggest an alternative?

Reply:
updated 🤝